### ISSUE TITLE
wadl:param are not getting generated for GET methods

### ISSUE THREAD

**Issue by alexander00001** [stoicflame/enunciate#406]
Hi everybody,

I'm trying to generate some wadl file from my resteasy java code. The wadl file and the documentation is generated. In the wadl the wadl:param tags are missing when the method is GET.

Example:

```java
    /**
     * Am i alive?
	 *
     * @param someId whatever.
     * @return Returns a HTTP code 200 after a successful modification.  
     */
    @GET
    @Path("alive/{someId}")
    @Produces(MediaType.TEXT_PLAIN)
    @ACL(allowAll = true)
    public Response onTest(@PathParam("someId") final Long someId) {
        return Response.ok().build();
    }
```

leads to this fragment in my wadl file:

```xml
<wadl:resource path="alive/{someId}">
  <wadl:method name="GET">
	<wadl:doc><![CDATA[Am i alive?]]></wadl:doc>
	<wadl:request>
	</wadl:request>
	<wadl:response>
	  <wadl:doc><![CDATA[Returns a HTTP code 200 after a successful modification.]]></wadl:doc>
	  <wadl:representation mediaType="text/plain"/>
	</wadl:response>
  </wadl:method>
</wadl:resource>
 ```

I would like to have something like this:

```xml
<wadl:resource path="{mandant}/scheduler/alive/{someId}">
  <wadl:method name="GET">
	<wadl:doc><![CDATA[Am i alive?]]></wadl:doc>
	<wadl:request>
		<wadl:param name="someId" style="path">
			<wadl:doc><![CDATA[whatever]]></wadl:doc>
        </wadl:param>
	</wadl:request>
	<wadl:response>
	  <wadl:doc><![CDATA[Returns a HTTP code 200 after a successful modification.]]></wadl:doc>
	  <wadl:representation mediaType="text/plain"/>
	</wadl:response>
  </wadl:method>
</wadl:resource>
```

Why is the wadl:param tag not generated? If the method is PUT then it is generated inside a "wadl:representation" tag.

I'm using enunciate 2.3.

Thanks in advance for your help.
Alex

**Comment by stoicflame** [comment]
The WADL spec doesn't define `style="path"`. See [section 2.12.2](https://www.w3.org/Submission/wadl/#x3-270002.12.2). The fact that path params were showing up under the `representation` tag was a bug, fixed at ac28545.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
